### ISSUE TITLE
Adaptive does a full bed probe on a 40x40x40mm test cube

### ISSUE THREAD

**Issue by foureight84** [kyleisah/Klipper-Adaptive-Meshing-Purging#83]
**Describe the issue**
I have a fairly warped bed that's bowed (convexed) in the middle. Using KAMP has been working fairly well except today when I tried printing a test 40x40x40mm cube in vase mode to do some extrusion multiplier calibration and the adaptive meshing performed a whole bed probe instead of the small area around the print. The bed besh size is set at 7x7 and has been working until this point in the refined area. However, 7x7 over the span of 220x220 on a bowed bed with a 0.15 first layer resulted in the nozzle dragging across the bed and ultimately blocking the nozzle. Just to give an example of how bad the bed bows once heated, I used to do a 30x30 bed mesh and load that profile before each print, and the bed mesh calibration would need to be rerun every 10 or so prints just to make sure it's accurate.

**What slicer are you using?**
Prusaslicer 2.5.2

**Troubleshooting**
I have tried smaller prints and KAMP seems to work fine.

**Screenshots**

**Klipper log file:**
[klippy (4).log](https://github.com/kyleisah/Klipper-Adaptive-Meshing-Purging/files/11402927/klippy.4.log)

**Sliced Gcode**

Please add sliced gcode file you are using here...
<details>
<summary>cube-40-40-40_0.2mm_PETG_MEGA0_41m.gcode</summary>
[https://pastebin.com/sZQhXYjw](https://pastebin.com/sZQhXYjw)
</details>

**Additional context**

**Comment by foureight84** [comment]
Quick update on this. Looks like there's an issue that might be caused by settings changes in the slicer. Although The only thing I've changed is related to extrusion width and first-layer height. I went back to reprint old gcodes sliced prior to these changes and Adaptive meshing is working as expected. The custom gcode with adaptive mesh settings is the same in both. Still trying to debug this on my end. I'll leave this open for now to update with additional information as I find them.
